ui/page/main_page.go

```diff
@@ -470,6 +480,33 @@ func (mp *MainPage) LayoutUSDBalance(gtx layout.Context) layout.Dimensions {
 	)
 }
 
+func (mp *MainPage) totalUSDBalance(gtx layout.Context) layout.Dimensions {
+	if mp.isBalanceHidden {
+		hiddenText:= mp.Theme.Body2("$******")
+		return hiddenText.Layout(gtx)
+	}
+	return mp.Theme.Body2(mp.totalBalanceUSD).Layout(gtx)
+}
```

Comment:
The USD balance shouldn't show asteriks, it should be removed entirely if hide balance is enabled